css/styles.css

```diff
@@ -58,4 +58,22 @@
   border: solid;
 }
 
+.control-me::after {
+  content: "";
+  font-size: 100px;
+}
```

Comment:
I think you don't need this ruleset

Reply:
You can instead use the display: none property to hide the answer and set the display to "block" when the checkmark is set.